jsonrpc/types.go

```diff
@@ -45,50 +45,103 @@ func toTransaction(t *types.Transaction, b *types.Block, txIndex int) *transacti
 	}
 }
 
+type uncle struct {
+	ParentHash        types.Hash    `json:"parentHash"`
+	Sha3Uncles        types.Hash    `json:"sha3Uncles"`
+	Miner             types.Address `json:"miner"`
+	StateRoot         types.Hash    `json:"stateRoot"`
+	TxRoot            types.Hash    `json:"transactionsRoot"`
+	ReceiptsRoot      types.Hash    `json:"receiptsRoot"`
+	LogsBloom         types.Bloom   `json:"logsBloom"`
+	Difficulty        argUint64     `json:"difficulty"`
+  TotalDifficulty   argUint64     `json:"totalDifficulty"`
+  Size              argUint64     `json:"size"`
```

Comment:
Minor nitpick: Please format this file

Reply:
Thanks @zivkovicmilos , now that I know out how to apply proper formatting this won't happen again 😄